test/src/api/commands/web-element/testisPresent.js

```diff
@@ -0,0 +1,120 @@
+const assert = require('assert');
+const {WebElement} = require('selenium-webdriver');
+const MockServer = require('../../../../lib/mockserver.js');
+const CommandGlobals = require('../../../../lib/globals/commands-w3c.js');
+const common = require('../../../../common.js');
+const Element = common.require('element/index.js');
+
+describe('element().isPresent() command', function () {
+  before(function (done) {
+    CommandGlobals.beforeEach.call(this, done);
+  });
+
+  after(function (done) {
+    CommandGlobals.afterEach.call(this, done);
+  });
+
+  it('test .element().isPresent() present', async function () {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/displayed',
```

Comment:
`/displayed` is not the correct endpoint for this. In fact, no endpoints are required for the `isPresent()` command.

Reply:
will this be right endpoint for this   ("/session/13521-10219-202/elements")